buildSrc/src/main/groovy/openhouse.iceberg-conventions.gradle

```diff
@@ -6,6 +6,6 @@ dependencies {
   implementation('org.apache.iceberg:iceberg-bundled-guava:' + icebergVersion)
   implementation('org.apache.iceberg:iceberg-data:' + icebergVersion)
   implementation('org.apache.iceberg:iceberg-core:' + icebergVersion)
-
-  testImplementation('org.apache.iceberg:iceberg-common:' + icebergVersion)
+  implementation('org.apache.iceberg:iceberg-common:' + icebergVersion)
```

Comment:
So iceberg common is needed as `implementation` with ADLS integration?

Reply:
As of right now, yes. If I do the submodules later I may reevaluate